ui/components/ui/disclosure/disclosure.js

```diff
@@ -47,24 +47,32 @@ const renderSummaryByType = (variant, title, size) => {
   }
 };
 
-const Disclosure = ({ children, title, size, variant }) => {
+const Disclosure = ({
+  children,
+  isScrollToBottomOnOpen,
```

Comment:
Should we default `isScrollToBottomOnOpen` to true to keep it enabled by default ?

Reply:
hi @jpuri, we should not since there is no need to open the Disclosure with scroll. Also, there is a potential bug with the initial implementation with scrollToBottom on. It scrolls to the bottom of the Disclosure component with no buffer. This means, if there is plenty of space below the Disclosure component, it may scroll underneath the component and skip over the entirety of the Disclosure content. Luckily, for the initial instance and only other instance of Disclosure, transaction-list-item-details.component.js, there is not much content below. Thus, when it scrolls down, the Disclosure component still can be seen.

Reply:
hi @jpuri, we should not since opening the Disclosure with scroll is unnecessary. 

Also, there is a potential bug with the initial implementation with scrollToBottom turned on. It scrolls to the bottom of the Disclosure component with no buffer. This means that if there is plenty of space below the Disclosure component, it may scroll underneath the component and skip over the entirety of the Disclosure content. This bug can be seen as reported by this issue. 

Luckily, for the initial instance and only other instance of Disclosure, transaction-list-item-details.component.js, there is little content below. Thus, the Disclosure component can still be seen when it scrolls down and seemingly works fine. I question why the original UI desired to scroll down to begin with.